guides/common/modules/con_managing-dns-using-smart-proxy.adoc

```diff
@@ -12,13 +12,13 @@ Other providers provide more direct integration, such as `dns_infoblox` for http
 ifdef::orcharhino[]
 * `dns_dnscmd` {endash} Static DNS records in Microsoft Active Directory.
 endif::[]
-* `dns_infoblox` {endash} For more information, see {ProvisioningDocURL}Using_Infoblox_as_DHCP_and_DNS_Providers_provisioning[Using Infoblox as DHCP and DNS Providers] in _{ProvisioningDocTitle}_.
+* `dns_infoblox` {endash} For more information, see xref:Installing_the_DHCP_Infoblox_Module_{context}.
 ifndef::satellite[]
 * `dns_libvirt` {endash} Dnsmasq DNS via libvirt API.
 For more information, see xref:configuring_dns_libvirt_{context}[].
 endif::[]
 * `dns_nsupdate` {endash} Dynamic DNS update using nsupdate.
-For more information, see {ProvisioningDocURL}Using_Infoblox_as_DHCP_and_DNS_Providers_provisioning[Using Infoblox as DHCP and DNS Providers] in _{ProvisioningDocTitle}_.
```

Comment:
This part triggered me when looking at it with @apinnick, because it's wrong. It escalated into a larger PR.